src/com/goide/inspections/unresolved/GoUnresolvedReferenceInspection.java

```diff
@@ -99,6 +99,10 @@ else if (parent instanceof GoRangeClause || parent instanceof GoRecvStatement) {
                 fixesList.add(new GoReplaceAssignmentWithDeclarationQuickFix(parent));
               }
             }
+
+            if (parent instanceof GoCallExpr) {
+              fixesList.add(0, new GoIntroduceFunctionFix(parent, name));
```

Comment:
1. who said you that order of fixes list matters?
2. if you want to add something at the start of list then invoke `add` when you created a list., this is ridiculous:
```
add(o1);
add(o2);
add(0, o3);
```